lib/modules/manager/glasskube/readme.md

```diff
@@ -0,0 +1,2 @@
+Renovate manager for Glasskube resources.
+Extract version data from Packages/ClusterPackages and repository data from PackageRepositories.
```

Comment:
```suggestion
Extract version data from Packages/ClusterPackages and repository data from PackageRepositories.

To use the `glaskube` manager you must set your own `fileMatch` pattern.
The `glaskube` manager has no default `fileMatch` pattern, because there is no common filename or directory name convention for Glaskube YAML files.
By setting your own `fileMatch` Renovate avoids having to check each `*.yaml` file in a repository for a Glaskube definition.
```